js/charts/chorddiagram.js

```diff
@@ -685,6 +685,61 @@ function chorddiagram() {
             .on('mouseover', _handleMouseOverFn.call(chart, tooltip, _local_svg))
             .on('mousemove', _handleMouseMoveFn.call(chart, tooltip, _local_svg))
             .on('mouseout', _handleMouseOutFn.call(chart, tooltip, _local_svg))
+            .on('click', function (d, i) {
+                if (isLiveEnabled) {
+                    broadcast.$broadcast('FlairBi:livemode-dialog');
```

Comment:
Better to write small functions...so that code can be readable and tested (will be written in future ) with ease